index.d.ts

```diff
@@ -10,8 +10,10 @@ declare global {
                 data: any;
                 cssIDs: string[];
             };
+            mustardCut: boolean;
             polyfilled: boolean;
             onPolyfilled: () => void;
+            queue: Array<(name: string, task: () => Promise<void>) => void>;
```

Comment:
Is this type correct? No args are passed to the queue function [when it is called in `htmlTemplate`](https://github.com/guardian/dotcom-rendering/pull/756/files#diff-5b02c564b94e434c72983f8b96284835R91)